openedx/core/djangoapps/programs/tests/test_utils.py

```diff
@@ -10,20 +11,22 @@
 from edx_oauth2_provider.tests.factories import ClientFactory
 from provider.constants import CONFIDENTIAL
 
+# This gets around an error about conflicting models documented at
+# https://code.djangoproject.com/ticket/22280. Importing the models from
+# `lms.djangoapps.certificates.models` raises a RuntimeError.
+from certificates.models import GeneratedCertificate  # pylint: disable=import-error
```

Comment:
@jimabramson good call, I've exposed the mode constants using a variable in the API module.